src/noos_inv/python.py

```diff
@@ -103,6 +104,8 @@ def package(ctx: Context, install: str | None = None) -> None:
         ctx.run("poetry build", pty=True)
     if install_type == InstallType.pipenv:
         ctx.run("pipenv run python -m build -n", pty=True)
+    if install_type == InstallType.uv:
+        ctx.run("uvx --from build pyproject-build --installer uv")
```

Comment:
**Suggestion:** Replace the direct command string with a formatted string to ensure consistency and maintainability, especially if additional options or configurations are needed in the future. [maintainability, importance: 5]
```suggestion
        ctx.run(f"uvx --from build pyproject-build --installer {InstallType.uv}")
```